tests/integration_tests/sqllab_tests.py

```diff
@@ -485,102 +485,6 @@ def test_pa_conversion_dict(self):
         self.assertEqual(len(data), results.size)
         self.assertEqual(len(cols), len(results.columns))
 
-    def test_sqllab_viz(self):
```

Comment:
nit: I think it's fine to leave in tests for deprecated endpoints, we can just delete them when they are eventually removed for good